spec/datetime.html

```diff
@@ -552,6 +552,33 @@ <h1>Temporal.DateTime.prototype.getTime ( )</h1>
   <emu-clause id="sec-temporal-datetime-abstract-ops">
     <h1>Abstract operations</h1>
 
+    <emu-clause id="sec-temporal-getepochfromparts" aoid="GetEpochFromParts">
+      <h1>GetEpochFromParts ( _year_, _month_, _day_, _hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanosecond_ )</h1>
+      <emu-alg>
+        1. Assert: _year_, _month_, _day_, _hour_, _minute_, _second_, _millisecond_, _microsecond_, and _nanosecond_ are integer Number values.
+        1. Assert: _month_ ≥ 1 and _month_ ≤ 12.
+        1. Assert: _day_ ≥ 1 and _day_ ≤ ! DaysInMonth(_year_, _month_).
+        1. Assert: ! ValidateTime(_hour_, _minute_, _second_, _millisecond_) is *true*.
+        1. Let _date_ be ! MakeDay(_year_, _month_, _day_).
```

Comment:
These links to MakeDay, MakeTime, and MakeDate in ecma262 do work, but I'm stumped how they are working without being added to biblio.json! Should they be added anyway?

Reply:
Links to 262 generally work - but not always. No clue why.